AxonDeepSeg/network_construction.py

```diff
@@ -1,83 +1,48 @@
-                                       normalizer_params={'scale':True, 'is_training':training_phase,
-                                                          'decay':bn_decay,'scope':'bn'},
-                                       weights_initializer = w_initializer, scope='convolution'
-                                      )
-        else:
-            net = tf.contrib.layers.conv2d(x, num_outputs=n_out_chan, kernel_size=k_size, stride=k_stride, 
-                                       activation_fn=tf.nn.relu, weights_initializer = w_initializer, scope='convolution'
-                                      )
-        net = tf.contrib.layers.dropout(net, keep_prob=keep_prob, is_training=training_phase)
-        tf.add_to_collection('activations',net)
-        return net
-    
-def downconv(x, n_out_chan, scope, 
-              w_initializer=tf.contrib.layers.xavier_initializer_conv2d(),
-              training_phase=True, activate_bn = True, bn_decay = 0.999):
-    
-    '''
-    Default data format is NHWC.
-    Initializers for weights and bias are already defined (default).
-    :param training_phase: Whether we are in the training phase (True) or testing phase (False)
-    '''
-    
-    with tf.variable_scope(scope):
-        if activate_bn == True:
-            net = tf.contrib.layers.conv2d(x, num_outputs=n_out_chan, kernel_size=5, stride=2, 
-                                       activation_fn=tf.nn.relu, normalizer_fn=tf.contrib.layers.batch_norm,
-                                       normalizer_params={'scale':True, 'is_training':training_phase,
-                                                          'decay':bn_decay, 'scope':'bn'},
-                                       weights_initializer = w_initializer, scope='convolution'
-                                      )
-        else:
-            net = tf.contrib.layers.conv2d(x, num_outputs=n_out_chan, kernel_size=5, stride=2, 
-                                       activation_fn=tf.nn.relu, weights_initializer = w_initializer, scope='convolution'
-                                      )
-        
-        tf.add_to_collection('activations',net)
-        return net
-    
-def upconv(x, n_out_chan, scope, 
+    net = Dropout(rate=1 - keep_prob)(net)
+
+    return net
+
+
+def downconv(x, filters, kernel_size=5, strides=2, activation='relu', kernel_initializer='glorot_normal',
+             activate_bn=True, bn_decay=0.999):
+    if activate_bn == True:
+
+        net = Conv2D(filters=filters, kernel_size=kernel_size, strides=strides, padding='same', activation=activation,
+                     kernel_initializer=kernel_initializer)(x)
+        net = BatchNormalization(axis=3, momentum=1 - bn_decay)(net)
+
+    else:
+
+        net = Conv2D(filters=filters, kernel_size=kernel_size, strides=strides, activation=activation,
+                     kernel_initializer=kernel_initializer, padding='same')(net)
+
+    return net
+
+
+"""def upconv(x, n_out_chan, scope, 
```

Comment:
I think these commented out pieces of code need to be removed

Reply:
Fixed in (328dcfc)